pinot-orc/src/test/java/org/apache/pinot/orc/data/readers/ORCRecordReaderTest.java

```diff
@@ -47,11 +51,13 @@
 public class ORCRecordReaderTest {
   private static final File TEMP_DIR = FileUtils.getTempDirectory();
   private static final File ORC_FILE = new File(TEMP_DIR.getAbsolutePath(), "my-file.orc");
+  private static final File MULTIVALUE_ORC_FILE = new File(TEMP_DIR.getAbsolutePath(), "mv-my-file.orc");
```

Comment:
Can you include this to be part of the same table/file as the other values?

Reply:
Also, would be good to add an explicit test for bytes.

Reply:
I believe I added bytes if this is what you are referring to: 
    FieldSpec yFieldSpec = new DimensionFieldSpec("y", FieldSpec.DataType.BYTES, true);

Reply:
I have not been able to set the ListColumnVector correctly, so I am using builder APIs to do so. While I can put my other values in my MV test, no apis are exposed to add multiple rows, so I think I should keep them separate.

Reply:
ok; we can check on options for that. For now, please do add more than one row in the MV test.